A suggestion for 3.2 (#117)
ietf-wg-aipref/drafts: draft-ietf-aipref-vocab.md

For example, a stated preference allowing a use does not
(on its own) imply a right to use it for that purpose;
likewise, a stated preference disallowing a use does not
(on its own) mean that the content cannot be used for that purpose.

## Applicability and Legal Effect {#legal}

This document provides a set of definitions for different categories of use,
plus a system for associating simple preferences to each
(allow, disallow, or no preference; see {{model}}).

The categories of use that are defined as part of the vocabulary
are not always clearly applicable or inapplicable to a particular system or application.
The universe of possible systems is far more complex
than any simple vocabulary is capable of describing.
That means that some discretion could be involved
in deciding whether a preference applies.

The expression of preferences might activate regulatory or legal consequences,
which has implications for entities that consume those preferences.
Their interpretation of the meaning of different terms
could have legal ramifications.
Different jurisdictions could reach subtly different conclusions
about the applicability of each category of use

> to specific applications.

Reviewer: This is probably most redundant, but there is still a useful point: how a category applies to a given application could be a judgment call and that might vary between jurisdictions (if there is a legal effect).

for variation between implementations.

## Enforcement

This specification does not provide any enforcement mechanism,
and conformance to it does not encompass whether
preferences are actually respected during data processing.

Other mechanisms
-- technical, legal, or otherwise --
might enforce stated preferences.

> As such they will determine the consequences for ignoring a stated preference.

Reviewer: ```suggestion
As such they will determine the consequences for following or not following a stated preference.
```

Preferences do not imply control, either in the positive or negative.
For example, a stated preference allowing a use does not

> (on its own) imply an unconditional right to use it for that purpose;

Reviewer: ```suggestion
(on its own) imply a right to use it for that purpose;
```

likewise, a stated preference disallowing a use does not

> (on its own) mean that the content can never be used for that purpose.

Reviewer: ```suggestion
(on its own) mean that the content cannot be used for that purpose.
```

> ## Enforcement
> 
> This specification does not provide any enforcement mechanism,
> and conformance to it does not encompass whether
> preferences are actually respected during data processing. 
> 
> Other mechanisms
> -- technical, legal, or otherwise --
> might enforce stated preferences.
> As such they will determine the consequences for ignoring a stated preference.
> Preferences do not imply control, either in the positive or negative.
> For example, a stated preference allowing a use does not 
> (on its own) imply an unconditional right to use it for that purpose;
> likewise, a stated preference disallowing a use does not
> (on its own) mean that the content can never be used for that purpose.

Author: ```suggestion
## Enforcement

This specification does not provide any enforcement mechanism,
and conformance to it does not encompass whether
preferences are actually respected during data processing.

Other mechanisms
-- technical, legal, or otherwise --
might enforce stated preferences.
As such they will determine the consequences for ignoring a stated preference.
Preferences do not imply control, either in the positive or negative.
For example, a stated preference allowing a use does not
(on its own) imply an unconditional right to use it for that purpose;
likewise, a stated preference disallowing a use does not
(on its own) mean that the content can never be used for that purpose.
```
